src/runtime/Context.h

```diff
@@ -24,6 +24,7 @@
 #include "runtime/GlobalObject.h"
 #include "runtime/RegExpObject.h"
 #include "runtime/StaticStrings.h"
+#include "debugger/Debugger.h"
```

Comment:
Minor suggestion, how about moving this header file into `Context.cpp`?

Reply:
Good catch! That is not necessary. Patch is updated.